dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -359,19 +359,29 @@ def type_into_selector(input_text, selector)
   @browser.execute_script 'dashboard.clientState.trackProgress(true, 1, 100, "hourofcode", appOptions.serverLevelId)'
 end
 
-Then /^I verify progress in the header of the current page is "([^"]*)" for level (\d+)/ do |test_result, level|
+Then /^I wait to see the progress header$/ do
   steps %{
     And I wait to see ".header_level_container"
-    And I wait for 10 seconds
+    And I wait for 5 seconds
+  }
```

Comment:
Is there anything for which we can wait until its visible rather than waiting 5 seconds here?